aiohttp/helpers.py

```diff
@@ -741,3 +742,7 @@ def set_result(fut, result):
 def set_exception(fut, exc):
     if not fut.done():
         fut.set_exception(exc)
+
+def json_dumps(obj):
+    # Return the most compact serialisation.
+    return json.dumps(obj, ensure_ascii=False, allow_nan=False, separators=(',', ':'))
```

Comment:
```
asd='превед я русский текст, как же фигово китайцам с юникодом. да уж, вот же шь у них проблемы то!!'
timeit json.dumps(asd, ensure_ascii=True)  # before patch
100000 loops, best of 3: 3.06 µs per loop

timeit json.dumps(asd, ensure_ascii=False)   # after patch
100000 loops, best of 3: 4.8 µs per loop
```

BUT

```
len(json.dumps(asd, ensure_ascii=True))  # before patch
457

len(json.dumps(asd, ensure_ascii=False))  # after patch
97
```

```

Reply:
So, this is VOLUME vs CPU trade-off. Don't know what to do next -- maybe some way to configure it globally ?